scripts/build_coverage.sh

```diff
@@ -8,7 +8,10 @@ mvn compile
 cd ..
 cd ./web
 mvn compile
+<<<<<<< HEAD
```

Comment:
Merge conflict.

Reply:
This conflict was fixed and pushed to repository again.